dll/win32/shell32/lang/en-GB.rc

```diff
@@ -228,7 +228,7 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
     ICON "", 14000, 16, 7, 32, 32, WS_VISIBLE
     EDITTEXT 14001, 58, 9, 170, 14, ES_LEFT
-    LTEXT "Type of file:", 14004, 8, 40, 55, 10
+    LTEXT "Type:", 14004, 8, 40, 55, 10
```

Comment:
Please check that any other shell32 translation file that would still use untranslated string here, uses the new modification.